src/python/pants/backend/python/target_types.py

```diff
@@ -633,15 +592,15 @@ def value_or_global_default(self, pex_binary_defaults: PexBinaryDefaults) -> boo
 
 class PexResolveLocalPlatformsField(TriBoolField):
     alias = "resolve_local_platforms"
+    removal_version = "2.24.0.dev0"
```

Comment:
The Pex `--resolve-local-platforms` option applies to both `--platform` and `--complete-platform` foreign targets despite what the CLI help says (`--resolve-local-platforms` existed before `--complete-platform` was introduced and I failed to update the help). Since `--platform` went away but `--complete-platform` did not, `--resolve-local-platforms` can still be useful. In fact Pants uses it internally for FAAS:
https://github.com/pantsbuild/pants/blob/876075026628f875b410282e9be09969dd401d7f/src/python/pants/backend/python/util_rules/faas.py#L506-L508

Reply:
Docs fix here: https://github.com/pex-tool/pex/pull/2759

Reply:
Example useful case here: https://github.com/jasonwbarnett/repro-pants-debugger-issue/issues/1